{
  "_args": [
    [
      {
        "raw": "node-exceptions@^2.0.2",
        "scope": null,
        "escapedName": "node-exceptions",
        "name": "node-exceptions",
        "rawSpec": "^2.0.2",
        "spec": ">=2.0.2 <3.0.0",
        "type": "range"
      },
      "C:\\Users\\Maynor\\source\\repos\\EDII-PROYECTOFINAL\\proyecto_final\\node_modules\\edge.js"
    ]
  ],
  "_from": "node-exceptions@>=2.0.2 <3.0.0",
  "_id": "node-exceptions@2.0.2",
  "_inCache": true,
  "_location": "/node-exceptions",
  "_nodeVersion": "8.0.0",
  "_npmOperationalInternal": {
    "host": "s3://npm-registry-packages",
    "tmp": "tmp/node-exceptions-2.0.2.tgz_1497264381965_0.15966099058277905"
  },
  "_npmUser": {
    "name": "virk",
    "email": "virk@adonisjs.com"
  },
  "_npmVersion": "5.0.0",
  "_phantomChildren": {},
  "_requested": {
    "raw": "node-exceptions@^2.0.2",
    "scope": null,
    "escapedName": "node-exceptions",
    "name": "node-exceptions",
    "rawSpec": "^2.0.2",
    "spec": ">=2.0.2 <3.0.0",
    "type": "range"
  },
  "_requiredBy": [
    "/edge.js"
  ],
  "_resolved": "https://registry.npmjs.org/node-exceptions/-/node-exceptions-2.0.2.tgz",
  "_shasum": "2612b45929b774e3669557f852b47507765acfe7",
  "_shrinkwrap": null,
  "_spec": "node-exceptions@^2.0.2",
  "_where": "C:\\Users\\Maynor\\source\\repos\\EDII-PROYECTOFINAL\\proyecto_final\\node_modules\\edge.js",
  "author": {
    "name": "amanvirk,adonisjs"
  },
  "bugs": {
    "url": "https://github.com/poppinss/node-exceptions/issues"
  },
  "config": {
    "commitizen": {
      "path": "./node_modules/cz-conventional-changelog"
    }
  },
  "dependencies": {},
  "description": "Extendable error class for nodejs to extend native errors",
  "devDependencies": {
    "coveralls": "^2.11.6",
    "cz-conventional-changelog": "^1.1.5",
    "japa": "^1.0.3",
    "japa-cli": "^1.0.1",
    "nyc": "^10.2.0",
    "standard": "^5.4.1"
  },
  "directories": {
    "test": "test"
  },
  "dist": {
    "integrity": "sha512-nILccFLvnaOBoKZZtWfZipn/hlThZOT6UJOLX6SA3yLMYPcvLTIF26PXx73sPnAg45p05iYFYw2jvayNHz4rDA==",
    "shasum": "2612b45929b774e3669557f852b47507765acfe7",
    "tarball": "https://registry.npmjs.org/node-exceptions/-/node-exceptions-2.0.2.tgz"
  },
  "gitHead": "357a1f4f4c4cce7ade0a185db6ea39ae3ca08ef8",
  "homepage": "https://github.com/poppinss/node-exceptions#readme",
  "keywords": [
    "exception",
    "node-errors",
    "custom-errors",
    "Error"
  ],
  "license": "MIT",
  "main": "index.js",
  "maintainers": [
    {
      "name": "virk",
      "email": "virk@adonisjs.com"
    }
  ],
  "name": "node-exceptions",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/poppinss/node-exceptions.git"
  },
  "scripts": {
    "coverage": "nyc report --reporter=text-lcov | coveralls",
    "lint": "standard",
    "posttest": "npm run coverage",
    "pretest": "npm run lint",
    "test": "nyc npm run test:local",
    "test:local": "japa",
    "test:win": "node ./node_modules/japa-cli/index.js"
  },
  "version": "2.0.2"
}
